### PR TITLE
feat: Streamline and rename chart

### PR DESCRIPTION
**Pull request by juanjjaramillo**
## Description

Streamline and rename chart to simplify customer onboarding

## Type of change

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] New feature / enhancement (non-breaking change which adds functionality)
- [ ] Security fix
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] Documentation has been updated
- [ ] This change requires changes in testing:
  - [ ] unit tests
  - [ ] E2E tests

### CODE REVIEW

**Review by ramkrishankumarN — APPROVED** [review]

**Comment by juanjjaramillo** [comment]
@ramkrishankumarN you should not approve a PR unless it is ready. I comment on Slack when PRs are ready for review

**Comment by ramkrishankumarN** [comment]
> @ramkrishankumarN you should not approve a PR unless it is ready. I comment on Slack when PRs are ready for review

Thanks for the heads up. I will look at the changes again

**Review by ramkrishankumarN — COMMENTED** [review]
We are renaming `folder charts/newrelic-agent-operator` to `charts/k8s-agent-operator`  and changing the main Chart.yaml, right? Hard to comprehend these kinds of diffs in github.

**Pull request merged**
